docs/template_plugin/tests/functional/skip_tests_config.cpp

```diff
@@ -59,6 +59,20 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*ReferenceExperimentalGPLayerTest.*bf16.*)",
         // CVS-64105
         R"(.*ReferenceGatherElementsTestNegative.*)",
+        // CVS-64017
+        R"(.*ReferenceGatherTest.*dType=i16.*)",
```

Comment:
Here do we have an issue with the reference implementation or with the plugin?

Reply:
In Gather-1 and Gather-7 cases, the exception log is like below, so the issue seems to be with the plugin. The exception is occurred after `Wait(InferRequest::WaitMode::RESULT_READY);` in `ie_infer_async_request_thread_safe_default.hpp`.

`C++ exception with description "Interpreter backend doesn't implement evaluate method for OP Gather" thrown in the test body.`

In Pad-1 case, the exception log is like below, so the issue seems to be with reference implementation. The exception is occurred in `CommonReferenceTest::LoadNetwork()` around `CNNNetworkNGraphImpl::createDataForResult()`.

`inference-engine/src/inference_engine/src/cnn_network_ngraph_impl.cpp:75 Parameter_44741 has zero dimension which is not allowed" thrown in the test body.`